### ISSUE TITLE
Address Potential Accessibility Concerns With Mobile/Off-canvas Navigation

### ISSUE THREAD

**Issue by jomurgel** [WebDevStudios/wd_s#449]
Several projects have turned up some WCAG 2.0 AA issues with the Aria labeling/association with the off-canvas navigation. 

Need to audit these issues and fix in core if possible, or determine if a new solution is required.

```
Web Content Accessibility Guidelines (WCAG) 2.0, Level A: 4.1.2 Name, Role, Value

The HTML element uses ARIA that depends on other other ARIA being declared earlier on in the markup. Howerver, the parent ARIA declaration is not present, breaking the relationship.
```

**Comment by coreymcollins** [comment]
I wouldn't mind removing the off-canvas menu completely, if at all possible. On GS, we use a single menu for desktop and mobile and just adjust the styles for mobile breakpoints.

It might be worth looking into what it would take to remove it completely but still be able to offer the same animation and smoothness on mobile with a reconfigured primary nav as the mobile nav.

Part of the idea of having the second menu was for clients who would want to use a different menu on mobile, but I don't know how common a thing that actually is. It'd be worth looking into that as well.

**Comment by coreymcollins** [comment]
It should be noted that I think some of, if not all, of the a11y issues were resolved in the grid PR and some of them came from the primary nav not having aria labels set while the off-canvas nav actually did.

We'll regroup as a team on this once the grid PR is merged in to see if there are any issues still cropping up and figure out how we want to move forward with the various menus.

**Comment by coreymcollins** [comment]
This no longer appears to be an issue – I'm not getting any errors on the off-canvas menu at this point.

Going to close it out!
